templates/ulimit.erb

```diff
@@ -27,3 +27,8 @@
 <% else -%><% unless @stack_soft_limit.nil? -%><%= @ulimit_user -%> soft stack <%= @stack_soft_limit %><% end -%>
 <% unless @stack_hard_limit.nil? -%><%= @ulimit_user -%> hard stack <%= @stack_hard_limit %><% end -%>
 <% end -%>
+
```

Comment:
You'll want to avoid the whitespace here since those 2 extra lines will show up for users that aren't specifying rtprio

Reply:
Thanks @tas50 fixed :-)